include/dts-functions.sh

```diff
@@ -1453,19 +1453,34 @@ main_menu_options(){
 
       # Check for Dasharo Firmware for the current platform, continue to
       # packages after checking:
-      check_for_dasharo_firmware
+      if check_for_dasharo_firmware; then
+        # check_for_dasharo_firmware prints some messages, so these echoes
+        # should be inside condition. FIXME: what if credentials have access to
+        # firmware, but check_for_dasharo_firmware will not detect any platform?
+        # According to check_for_dasharo_firmware it will return 1 in both
+        # cases which means that we cannot detect such case.
+        echo "Your credentials give access to:"
+        echo -n "Dasharo Pro Package (DPP): "
+        print_ok "YES"
+      else
+        echo "Your credentials give access to:"
+	echo -n "Dasharo Pro Package (DPP): "
```

Comment:
You can move out the common part, since you will print those messages regardless of the result of the function

Reply:
Check comments above.